### ISSUE TITLE
optimize handling of 'incomplete' plot-files

### ISSUE THREAD

**Issue by de-luxe** [de-luxe/burstcoin-jminer#21]
create internal lookup with real number of nonces for plot-files, were size and specified size from filename does not match, to prevent checker from checking empty buffer.

**Comment by k06a** [comment]
I am mining while generating some files. Looks like miner uses unfinished files...

**Comment by Blagodarenko** [comment]
XPlotter blocks file for monopoly access while plotting

**Comment by k06a** [comment]
Right now I am creating plot files in subdirectory and move them to disk root after completion.

**Comment by de-luxe** [comment]
Yes, thats what i would suggest @k06a , miner will also (try to) mine unfinished plotfiles, if they are within the specified plotFilePaths ... depending on plotter used this may lead to undefined behavior ...
